patches/@lerna+package-graph+4.0.0.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/node_modules/@lerna/package-graph/index.js b/node_modules/@lerna/package-graph/index.js
+index 804aae0..dc1684f 100644
```

Comment:
(copy-pasting my comment from https://github.com/aws/aws-cdk/pull/14150#issuecomment-819379809)

Can we put some kind of message to the console stating that the cdk package is being used for package graph analysis?